pkg/scheduler/plugins/loadaware/load_aware.go

```diff
@@ -146,9 +146,29 @@ func (p *Plugin) Filter(ctx context.Context, state *framework.CycleState, pod *c
 		return nil
 	}
 
+	allocatable, err := p.estimator.EstimateNode(node)
+	if err != nil {
+		klog.ErrorS(err, "Failed to EstimateNode", "node", node.Name)
+		return framework.NewStatus(framework.Error, err.Error())
```

Comment:
We should not return framework.Error here, we should return framework.Unschedulable or UnschedulableAndUnresolvable to ensure that Pod is assigned to nodes as much as possible.

Reply:
I changed this return to nil; if there is an error, filter will not work.